seriallist_darwin.go

```diff
@@ -72,23 +73,13 @@ func removeNonArduinoBoards(ports []OsSerialPort) []OsSerialPort {
 	arduino_ports = append(arduino_ports, other_ports...)
 
 	// additional remove phase
-	for i, port := range arduino_ports {
-		if strings.Contains(port.Name, "Blue") || strings.Contains(port.Name, "/cu") {
-			arduino_ports = removePortFromSlice(arduino_ports, i)
-		}
-	}
+	arduino_ports = Filter(arduino_ports, func(port OsSerialPort) bool {
+		return !string.Contains(port.Name, "Blue") && !strings.Contains(port.Name, "/cu")
```

Comment:
it should be just a typo, strings.Contains instead of string.Contains, I am going to verify it.